Makefile

```diff
@@ -16,6 +16,17 @@ TARGET   := alertmanager
 
 include Makefile.COMMON
 
+REV        := $(shell git rev-parse --short HEAD 2> /dev/null  || echo 'unknown')
+BRANCH     := $(shell git rev-parse --abbrev-ref HEAD 2> /dev/null  || echo 'unknown')
+HOSTNAME   := $(shell hostname -f)
```

Comment:
Humm, I guess that's fine, since Solaris is not a supported Go platform? Do you know how it behaves on FreeBSD?